operations/app/terraform/modules/function_app/main.tf

```diff
@@ -3,69 +3,39 @@ resource "azurerm_function_app" "pdi" {
   location                   = var.location
   resource_group_name        = var.resource_group_name
   app_service_plan_id        = var.app_service_plan
-  storage_account_name       = var.sa_data_name
-  storage_account_access_key = var.sa_data_access_key
   https_only                 = true
   os_type                    = "linux"
   version                    = "~3"
   enable_builtin_logging     = false
+  storage_account_name       = var.sa_functionapps.name
+  storage_account_access_key = var.sa_functionapps.primary_access_key
 
   app_settings = {
     # Use the CDC DNS for everything; they have mappings for all our internal
     # resources, so if we add a new resource we'll have to contact them (see
     # prime-router/docs/dns.md)
-    "WEBSITE_DNS_SERVER" = "168.63.129.16"
-
-    # "DOCKER_REGISTRY_SERVER_URL"      = var.container_registry_login_server
-    # "DOCKER_REGISTRY_SERVER_USERNAME" = var.container_registry_admin_username
-    # "DOCKER_REGISTRY_SERVER_PASSWORD" = var.container_registry_admin_password
-
-    # With this variable set, clients can only see (and pull) signed images from the registry
-    # First make signing work, then enable this
-    # "DOCKER_CONTENT_TRUST" = 1
+    WEBSITE_DNS_SERVER = "168.63.129.16"
```

Comment:
we should remove the comment above this, as we're explicitly NOT using CDC DNS here